### DISCUSSION TITLE
Embedded systems content

### DISCUSSION THREAD

**Discussion by Batto1** [dailydotdev/daily#1152] [Feature Requests]
I guess embedded software engineers count as developers right 😅? I'd love to set preference and see embedded software related content such as C, RTOS, new embedded devices etc. in the newsfeed, too.

**Comment by idoshamun** [comment]
Hi @Batto1,

You're most definitely right! Feel free to suggest sources that are relevant.
For more info: https://docs.daily.dev/docs/for-content-creators/suggest-new-source
